Source/WTF/wtf/text/ASCIILiteral.h

```diff
@@ -63,6 +63,15 @@ class ASCIILiteral final {
         : ASCIILiteral()
     { }
 
+WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN
+    template<size_t length>
+    consteval ASCIILiteral(const char (&literal)[length])
```

Comment:
Why is this needed? We normally use `_s` instead.

Reply:
It saves typing out `_s` in the places where you pass a string literal to a function that takes an ASCIILiteral, which makes this patch smaller. 

It's type-safe and bounds-safe, so probably OK not to require `_s`?

Reply:
Maybe OK as a way to reduce patch size short term but we were working towards enforcing ASCIILiteral and _s everywhere, thus my worry.